npm/eslint-plugin-dev/lib/index.js

```diff
@@ -300,43 +300,17 @@ module.exports = {
         '@cypress/dev/skip-comment': 'error',
       },
       overrides: [{
-        files: '*.tsx',
+        files: '*.spec.tsx',
         parser: '@typescript-eslint/parser',
-        parserOptions: {
-          ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
-          sourceType: 'module', // Allows for the use of imports
-          ecmaFeatures: {
-            jsx: true, // Allows for the parsing of JSX
-          },
-        },
-        settings: {
-          react: {
-            version: 'detect', // Tells eslint-plugin-react to automatically detect the version of React to use
-          },
-        },
         plugins: [
           '@typescript-eslint',
           'react',
         ],
         rules: {
           'no-unused-vars': 'off', // avoid interface imports to be warned against
```

Comment:
For some reason we have some linting problems with `spec.tsx` files - there are no other `spec.tsx` files in the code-base. I think we can fix this by updating the version of eslint and the other deps, but a lot of things broke when I tried, so I gave up and just left it as this for now.